package.json

```diff
@@ -33,12 +33,12 @@
   "dependencies": {
     "@babel/core": "7.0.0-beta.47",
     "@fortawesome/free-solid-svg-icons": "^5.1.0",
-    "@patternfly/patternfly-next": "1.0.79",
-    "@patternfly/react-charts": "1.1.0",
-    "@patternfly/react-core": "1.36.0",
-    "@patternfly/react-icons": "2.8.0",
+    "@patternfly/patternfly-next": "1.0.106",
```

Comment:
Will we need Insights to bump any of their dependencies for this? If so, do we open an issue on that?

Reply:
I don't believe there are any breaking changes. I'm mainly interested in the bug fixes. I'll open an issue if we encounter any conflicts.

Reply:
Theoretically, the changes in `1.0.106` should be backward compatible with `1.0.79` but it always a good idea to make sure it doesn't actually break the UI.

Anyway, LGTM. :+1: